win32/bin/search.pl

```diff
@@ -1,6 +1,10 @@
 #!/usr/local/bin/perl -w
-'di';
-'ig00';
```

Comment:
I'm not sure what is the value of these two isolated strings?
Are they required? testing something? was it just an internal joke that we can clear?